addons/mail/static/src/models/thread_cache/thread_cache.js

```diff
@@ -329,6 +313,19 @@ function factory(dependencies) {
             compute: '_computeFetchedMessages',
             dependencies: ['threadMessages'],
         }),
+        /**
+         * Determines whether the thread cache should load messages.
+         */
+        hasToLoadMessages: attr({
+            compute: '_computeHasToLoadMessages',
+            dependencies: [
+                'isLoaded',
+                'isLoading',
+                'thread',
+                'threadIsTemporary',
+                'threadViewers',
+            ],
+        }),
```

Comment:
Would precise it's a dummy field, and this is used to auto-load messages.

Reply:
In this case it's a real field, which holds a value that has a meaning ("is going to load soon"). The dummy field is the corresponding compute.